src/feedback/views/public_class_view.py

```diff
@@ -25,7 +25,7 @@ def barcodedrop(request):
             if next_state is None:
                 response_message = "Nachster Status ungultig."
             else:
-                time_offset = datetime.date.today() - datetime.timedelta(hours=1)
+                time_offset = datetime.date.today() - datetime.timedelta(minutes=1)
```

Comment:
datetime.date.today() is not what you want. datetime.datetime.now() has a time resolution and not only a date resulution.